packages/truffle/test/scenarios/commands/develop.js

```diff
@@ -15,7 +15,7 @@ const formatLines = lines =>
     .map(line => `\t---truffle develop log---\t${line}`)
     .join("\n");
 
-describe("truffle develop", function () {
+describe.only("truffle develop", function () {
```

Comment:
you will want to get rid of this `.only`

Reply:
I am sorry about that. I forgot to remove it after I tested it locally.